ansible/deploy-openstack-config.yml

```diff
@@ -118,25 +118,6 @@
       vars:
         kayobe_environment_path: "{{ src_directory }}/kayobe-config/etc/kayobe/environments/{{ kayobe_config_environment }}"
 
-    # WORKAROUND:
-    # There is a race condition in Ansible that can result in this failure:
```

Comment:
It's described in the commit message but got lost in the PR.

> We set the config option as an environment variable rather than in
ansible.cfg in Kayobe configuration, to avoid a merge conflict on upgrade.